client/my-sites/media-library/test/list.jsx

```diff
@@ -3,209 +3,287 @@
  */
 
 /* eslint jest/expect-expect: ["error", { "assertFunctionNames": ["expectSelectedItems", "expect"] }] */
-import { mount } from 'enzyme';
-import { defer } from 'lodash';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
 import moment from 'moment';
+import { useState } from 'react';
 import { MediaLibraryList as MediaList } from '../list';
 import fixtures from './fixtures';
 
 /**
  * Module variables
  */
 const DUMMY_SITE_ID = 2916284;
-const mockSelectedItems = [];
-
-jest.mock( 'calypso/components/infinite-list', () =>
-	require( 'calypso/components/empty-component' )
-);
-jest.mock( 'calypso/my-sites/media-library/list-item', () =>
-	require( 'calypso/components/empty-component' )
-);
-jest.mock( 'calypso/my-sites/media-library/list-plan-upgrade-nudge', () =>
-	require( 'calypso/components/empty-component' )
-);
+
+jest.mock( 'calypso/my-sites/media-library/list-item', () => ( { media: mediaItem, onToggle } ) => (
+	<button
+		data-testid={ `list-item-${ mediaItem.ID }` }
```

Comment:
Nit: do we need a `data-testid` when we can use `getByRole()` because it's a `button`?

Reply:
I used `data-testid` to make it more clear it's a mocked element whereas with `getByRole` it's not necessarily obvious. You're right though, `getByRole` should work just fine as well.